cpp/util/etcd_delete_test.cc

```diff
@@ -221,6 +224,22 @@ TEST_F(EtcdDeleteTest, ErrorHandling) {
 }
 
 
+TEST_F(EtcdDeleteTest, ForceDelete) {
+  vector<pair<string, int64_t> > keys{make_pair("/one", 1),
+                                      make_pair("/two", 0)};
+  SyncTask sync(&pool_);
+
+  EXPECT_CALL(client_, Delete("/one", 1, _))
```

Comment:
I think this is failing because `Task::Return()` returns a `bool`, gmock noticed that and is trying to use that returned bool as the return value for the mocked function which is `void`

You probably want something like:
```c++
EXPECT_CALL(client_, Delete("/one", 1, _))
    .WillOnce(IgnoreResult(Invoke(bind(&Task::Return, _3, Status::OK))));
```